### ISSUE TITLE
Mark workspaces with favorite layout?

### ISSUE THREAD

**Issue by keeganstothert** [Leleat/Tiling-Assistant#214]
I like to organize my workspaces with separate projects and layouts. 

Any way to allow workspaces to be treated like a monitor and have their own favorite layout?

Thank you for this amazing extension btw, it made it possible for me to finally switch to linux full time

**Comment by Leleat** [comment]
I guess this would require the user to make the workspaces static, right? Don't think it would work with the dynamic workspaces?

**Comment by ecogit** [comment]
> 

Yes, to name workspaces and allocate applications to it, static is required
